.perlcriticrc

```diff
@@ -5,6 +5,7 @@ severity=1
 [BuiltinFunctions::ProhibitLvalueSubstr]
 [BuiltinFunctions::ProhibitReverseSortBlock]
 [BuiltinFunctions::ProhibitShiftRef]
+[BuiltinFunctions::ProhibitSleepViaSelect]
```

Comment:
I'm not sure we should add every random rule that we don't need TBH, since every extra rule it has to check surely slows down perlcritic incrementally?

Reply:
I can drop this one if you're not worried about us ever using it :)